Source/warningreporting/WarningReportingUnit.cpp

```diff
@@ -189,7 +189,7 @@ namespace WarningReporting {
 
     void WarningReportingUnit::UpdateEnabledCategories(const Core::JSON::ArrayType<Setting::JSON>& info)
     {
-        Core::SafeSyncType<Core::CriticalSection> guard(_adminLock);
+        _adminLock.Lock();
```

Comment:
You don't have to change this but this is an example where we now could move the Lock after the next statement, so take the lock a little shorter, but as influence will be minimal no need to change it